python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -97,8 +85,12 @@ class DcrAssembleCoaddConnections(pipeBase.PipelineTaskConnections,
 
     def __init__(self, *, config=None):
         super().__init__(config=config)
+        self.outputs.remove("coaddExposure")
```

Comment:
I assume these are part of AssembleCoaddConnections but this method does not output them, correct? If that's the case please add a comment here to the explaining why these two outputs are removed.